packages/orchestration/src/examples/auto-stake-it-tap-kit.js

```diff
@@ -20,7 +20,13 @@ const trace = makeTracer('AutoStakeItTap');
 /**
  * @typedef {{
  *   stakingAccount: ERef<OrchestrationAccount<any> & StakingAccountActions>;
- *   localAccount: ERef<OrchestrationAccount<{ chainId: 'agoric' }>>;
+ *   localAccount: ERef<
+ *     OrchestrationAccount<{
+ *       chainId: 'agoric-3';
```

Comment:
you could leave this alone by making only CosmosChainInfo have `chainId`. I think that's consistent with master.